karate-demo/src/test/java/demo/lighthouse/demo-07.feature

```diff
@@ -0,0 +1,9 @@
+Feature: browser automation 1
+
+  Scenario: lighthouse impl
+
+    Given lightHouseWrapper 'https://github.com/login' accessibility,performance
+    * print LHR.categories.accessibility.score
+    * print LHR.categories.performance.score
+    #* match LHR.categories.accessibility.score == 0.96
+    #* match LHR.categories.performance.score == 1
```

Comment:
possible further work to check for > , < threshold values on accessibility / performance etc scores. Additionally, we can generate complete report.